src/meltano/cli/__init__.py

```diff
@@ -4,9 +4,11 @@
 import logging
 import os
 import sys
+from typing import TYPE_CHECKING
 
 from meltano.core.logging import setup_logging
 from meltano.core.project import ProjectReadonly
+from meltano.core.tracking.contexts.exception import ExceptionContext
```

Comment:
fyi - the pre-commit hooks and flake8 linters are gonna complain about this unused import. If its a false positive and you're using it as a magic import you can throw a `# noqa: F401` on there.